src/components/LifeEventPicker.tsx

```diff
@@ -1,54 +1,287 @@
-      <Typography style={{ marginBottom: 20, color: 'blue' }}>
-        Birth | Residence | Marriage | Death | Any
+      <Typography style={{ marginBottom: 20 }}>
+        <Link onClick={addBirthTextField}>Birth</Link>
+        {' | '}
+        <Link onClick={addResidenceTextField}>Residence</Link>
+        {' | '}
+        <Link onClick={addMarriageTextField}>Marriage</Link>
+        {' | '}
+        <Link onClick={addDeathTextField}>Death</Link>
+        {' | '}
+        <Link onClick={addAnyEventTextField}>Any</Link>
       </Typography>
-      <Grid container spacing={2}>
-        <Grid item xs={12} sm={6}>
-          <TextField
-            autoComplete="fname"
-            name="birthplace"
-            variant="outlined"
-            required
-            fullWidth
-            id="birthplace"
-            label="Birthplace"
-            autoFocus
-          />
-        </Grid>
-        <Grid item xs={6} sm={3}>
-          <TextField
-            variant="outlined"
-            required
-            fullWidth
-            id="startYear"
-            label="Year Start"
-            name="startYear"
-          />
-        </Grid>
-        <Grid item xs={6} sm={3}>
-          <TextField
-            variant="outlined"
-            required
-            fullWidth
-            id="endYear"
-            label="Year End"
-            name="endYear"
-          />
-        </Grid>
-      </Grid>
+
+      <div>
+        {state.birth ? (
+          <Grid container spacing={2} style={{ marginBottom: 20 }}>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                name="birthplace"
+                variant="outlined"
+                required
+                fullWidth
+                id="birthplace"
+                label="Birthplace"
+                autoFocus
+              />
+            </Grid>
+            <Grid item xs={6} sm={3}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                id="birthplaceStartYear"
+                label="Year Start"
+                name="birthplaceStartYear"
+              />
+            </Grid>
+            <Grid item xs={6} sm={3}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                id="birthplaceEndYear"
+                label="Year End"
+                name="birthplaceEndYear"
+              />
+            </Grid>
+          </Grid>
+        ) : (
+          ''
+        )}
+      </div>
+      {/* ------------------- */}
+      <div>
+        {state.residence ? (
+          <Grid container spacing={2} style={{ marginBottom: 20 }}>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                name="residence"
+                variant="outlined"
+                required
+                fullWidth
+                id="residence"
+                label="Residence"
+                autoFocus
```

Comment:
I'd have to play with it, but it seems strange to have multiple fields set with `autoFocus`.

Reply:
It looks like that with autofocus here - 
after you click on a life event the input cursor will move into that text field automatically . Without autofocus, no text field is selected (eg. death text field)

![autofocus](https://user-images.githubusercontent.com/49572089/84715469-687e9b80-af25-11ea-9ffd-a60d9da25c21.gif)